src/Umbraco.Core/Configuration/Models/SecuritySettings.cs

```diff
@@ -116,4 +116,14 @@ public class SecuritySettings
     /// </summary>
     [DefaultValue(StaticAllowConcurrentLogins)]
     public bool AllowConcurrentLogins { get; set; } = StaticAllowConcurrentLogins;
+
+    /// <summary>
+    ///     Gets or sets a value of the backoffice host URI.
+    /// </summary>
+    public Uri? BackOfficeHost { get; set; }
+
+    /// <summary>
+    ///     Gets or sets a value indicating whether to allow concurrent logins.
```

Comment:
```suggestion
    ///     The path to use for authorization callback. Will be appended to the BackOfficeHost.
```

Would also be good to make this non-nullable with a default value of `/umbraco` ... instead of having that default value hardcoded in `BackOfficeApplicationManager` (the currently only consumer of this configuration value).